### ISSUE TITLE
merge.exe should include all optional files and columns

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#99]
```
Agencies are being asked to use merge.exe for all files, but some files and 
columns aren't transferred to the merged file.  Here are some results from 
San Diego's merge:

stop_times.txt
optional column timepoint was dropped by the tool

trips.txt
optional columns wheelchair_accessible and trip_bikes_allowed were dropped 
by the tool

optional files
Files transfers.txt and feed_info.txt were dropped completely by the merge 
tool.

```

Original issue reported on code.google.com by `devin.br...@gmail.com` on 13 Oct 2008 at 3:35

**Comment by GoogleCodeExporter** [comment]
Original comment by `tom.brow...@gmail.com` on 5 Nov 2008 at 7:53
- Added labels: **App-Merge**, **Language-Python**

**Comment by GoogleCodeExporter** [comment]
```
As of r1271 transfers.txt is preserved.
Optional columns in trips.txt are preserved as of r969, though I don't think 
there is 
a test.

Preserving arbitrary columns in stop_times is a little harder. Preserving files 
that 
are identical in both inputs GTFS zips sounds like a good idea and not super 
difficult.
```

Original comment by `tom.brow...@gmail.com` on 25 Jan 2010 at 5:04
- Added labels: **Difficulty-Moderate**

**Comment by GoogleCodeExporter** [comment]
```
I agree that feed_info.txt should also be merged, and the latest version of the 
released code (1.2.11) does not preserve transfers.txt.
```

Original comment by `pdxclank...@gmail.com` on 5 Jul 2012 at 4:39

**Comment by GoogleCodeExporter** [comment]
```
Moved to https://github.com/google/transitfeed/issues/99
```

Original comment by `bdfer...@google.com` on 7 Oct 2014 at 7:58
- Changed state: **WontFix**
